### ISSUE TITLE
uppAliasOnly is deprecated; use useAlias

### ISSUE THREAD

**Issue by judgej** [academe/omnipay-datatrans#5]
It's used in a few places, so needs to be handled differently.

```text
     * @var string value to request a card alias only (uppAliasOnly data item)
./src/Interfaces/Constants.php
        $data[] = $this->getAmountInteger() ?: 'uppAliasOnly';
./src/Message/AbstractRequest.php
            $data['uppAliasOnly'] = Gateway::CARD_ALIAS_ONLY;
./src/Message/AbstractRedirectRequest.php
```

The new `useAlias` is also used, so it may well be covered, but we need to be sure:

```text
                $data['useAlias'] = Gateway::USE_ALIAS;
./src/Message/AbstractRedirectRequest.php
            'useAlias'   => 'yes'
./src/Message/TokenizeRequest.php
            'useAlias' => 'yes',
            'useAlias' => 'yes',
./tests/Message/TokenizeRequestTest.php
```

**Comment by judgej** [comment]
Now a zero amount is treated like any other amount in the UPP hash, rather than being replaced by the `uppAliasOnly` literal string.

If the amount is zero, then the `useAlias` flag is automatically set with the assumption that a card alias is being requested. This should work with other payment type aliases too, such as the PayPal `orderId`.

We set `useAlias` to `yes`, as that appears most often in documentation. In [some places](https://www.datatrans.ch/alias-tokenization) it is specified as `true`, and other places `1`. They all seem to work. For now.

**Comment by judgej** [comment]
Released as version 3.0.4

Not back-ported to 2.x. This can be done if there is a demand.
